### PR TITLE
Added the "Drupal Console" project

### PR DESCRIPTION
**Pull request by javiereguiluz**
@jmolivas and @dmouse, please review that the information is accurate. Thanks!

### CODE REVIEW

File: projects/drupalconsole.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Drupal Console
+url: http://drupalconsole.com/
+dependencies: []
+components:
```
Comment (jmolivas) [inline projects/drupalconsole.yml]: @javiereguiluz You are missing the `Config` component

**Comment by jmolivas** [comment]
@javiereguiluz: Besides the note I left, related to the missing component, the rest looks great.

Thanks for adding the project.

**Comment by dmouse** [comment]
:+1:  thanks @javiereguiluz !

**Comment by javiereguiluz** [comment]
Merged! Thanks @jmolivas and @dmouse for creating this fantastic project based on Symfony components!

**Pull request closed without merging**